src/sider/dialogmoter/motebehov/MotebehovKvittering.tsx

```diff
@@ -140,62 +169,170 @@ export function MotebehovArbeidsgiverKvittering({
 
 interface MotebehovKvitteringInnholdArbeidsgiverUtenMotebehovProps {
   ledereUtenInnsendtMotebehov: NarmesteLederRelasjonDTO[];
+  skjemaType: MotebehovSkjemaType | null;
 }
 
 export const MotebehovKvitteringInnholdArbeidsgiverUtenMotebehov = ({
   ledereUtenInnsendtMotebehov,
+  skjemaType,
 }: MotebehovKvitteringInnholdArbeidsgiverUtenMotebehovProps) => (
   <>
     {ledereUtenInnsendtMotebehov.map(
       (leder: NarmesteLederRelasjonDTO, index: number) => {
         const ikonAltTekst = `Arbeidsgiver ${arbeidsgiverNavnEllerTomStreng(
           leder.narmesteLederNavn
-        )} ${ikonAlternativTekst(undefined)}`;
+        )} ${ikonAlternativTekst(skjemaType)}`;
         return (
           <MotebehovKvitteringInnhold
             key={index}
             ikonAltTekst={ikonAltTekst}
-            tekst={composeArbeidsgiverSvarText(leder.narmesteLederNavn)}
+            tekst={composeArbeidsgiverSvarText(
+              leder.narmesteLederNavn,
+              skjemaType
+            )}
           />
         );
       }
     )}
   </>
 );
 
-export default function MotebehovKvittering() {
-  const { data: motebehov } = useMotebehovQuery();
-  const { currentLedere } = useLedereQuery();
-
-  const { tilfellerDescendingStart, latestOppfolgingstilfelle } =
-    useOppfolgingstilfellePersonQuery();
-
-  const aktiveMotebehovSvar = motebehovFromLatestActiveTilfelle(
-    motebehov?.sort(sorterMotebehovDataEtterDato),
-    latestOppfolgingstilfelle
+function MotebehovInCurrentTilfelle({
+  motebehovInTilfelle,
+  oppfolgingstilfelle,
+}: {
+  motebehovInTilfelle: MotebehovVeilederDTO[];
+  oppfolgingstilfelle: OppfolgingstilfelleDTO;
+}) {
+  const motebehovArbeidstaker = motebehovInTilfelle.find(
+    isArbeidstakerMotebehov
   );
-
-  const ledereUtenInnsendtMotebehov = ledereUtenMotebehovsvar(
-    currentLedere,
-    motebehov,
-    tilfellerDescendingStart || []
+  const motebehovArbeidsgiver = motebehovInTilfelle.find(
+    (motebehov) => !isArbeidstakerMotebehov(motebehov)
   );
 
+  function findSkjemaType(
+    motebehov: MotebehovVeilederDTO | undefined
+  ): MotebehovSkjemaType | null {
+    if (motebehovArbeidstaker && motebehovArbeidsgiver) {
+      return motebehov?.skjemaType ?? null;
+    } else if (motebehovArbeidstaker && !motebehovArbeidsgiver) {
+      return motebehovArbeidstaker?.skjemaType;
+    } else if (motebehovArbeidsgiver && !motebehovArbeidstaker) {
+      return motebehovArbeidsgiver?.skjemaType;
+    } else {
+      return null;
+    }
+  }
```

Comment:
Dersom én av aktørene har meldt/svart behov innenfor nåværende oppfølgingstilfelle, så vil vi vise at den andre parten ikke har meldt. Da har vi ikke noe skjematype å gå på for å generere teksten, så vi "låner" skjematypen fra den aktøren som faktisk har svart for å vise dette.